app/konnect/dev-portal/audit-logging/index.md

```diff
@@ -0,0 +1,17 @@
+---
+title: Audit Logging in Dev Portal
+badge: enterprise
+content_type: concept
+---
+
+{% include_cached /md/konnect/audit-logging/audit-log-overview.md %}
```

Comment:
I edited this so it's a bit more generic, like "By tracking audit logs..." so it would apply to both. Currently, I don't know of a way to make that bit specific to the feature area and still reuse for both pages. I wanted to reuse it because 99% of the content was the exact same, so it helps us keep both up-to-date and reduce translation costs in the future.

Reply:
You can create conditional includes here is a random [example](https://github.com/Kong/docs.konghq.com/blob/main/app/_includes/md/admin-listen.md)